app/assets/javascripts/admin/order_cycles/controllers/order_cycles_controller.js.coffee

```diff
@@ -6,6 +6,13 @@ angular.module("admin.orderCycles").controller "OrderCyclesCtrl", ($scope, $q, C
   $scope.involvingFilter = 0
   $scope.scheduleFilter = 0
```

Comment:
Hi Adam, welcome! These two lines above are redundant as they are inside resetSelectFilters which is called just below, isn't it?